npm/css-src/sass/_header.scss

```diff
@@ -58,3 +57,19 @@ $govuk-header-link: govuk-colour('white');
     text-decoration: none;
   }
 }
+
+@media (max-width: 48.06249em) {
```

Comment:
How did you derive the max width value?

Reply:
Because on GDS, they have a CSS rule for their header which states that when the screen width is over `48.0625em`,  the header should not display the content as vertically-stacked blocks and instead the content should be divided horizontally into sections
![image](https://user-images.githubusercontent.com/55027191/149159224-52c2a031-d232-4d91-b976-32996fad482d.png)

The CSS I wrote is just the opposite, any screen with a width of `48.06249em` and under, should no longer display content divided into horizontal sections and should be in blocks; I feel this this way reads better.